networkMiner/client.go

```diff
@@ -8,14 +8,13 @@ import (
 	"net"
 	"time"
 
-	"github.com/pegnet/pegnet/balances"
-
 	"github.com/FactomProject/factom"
+	"github.com/pegnet/pegnet/balances"
 	"github.com/pegnet/pegnet/common"
 	"github.com/pegnet/pegnet/mining"
 	"github.com/pegnet/pegnet/opr"
 	log "github.com/sirupsen/logrus"
-	"github.com/zpatrick/go-config"
+	config "github.com/zpatrick/go-config"
```

Comment:
My IDE forces me to put the `config` first, or the auto import keeps removing it.